lang/cs/Org.Apache.REEF.Client/AzureBatch/AzureBatchDotNetClient.cs

```diff
@@ -75,7 +81,9 @@ public void Submit(JobRequest jobRequest)
             var configModule = AzureBatchClientConfiguration.ConfigurationModule;
             string jobId = jobRequest.JobIdentifier;
             string commandLine = GetCommand(jobRequest.JobParameters);
-            _batchService.CreateJob(jobId, null, commandLine);
+            string jarPath = _jobJarMaker.CreateJobSubmissionJAR(jobRequest);
+            Uri blobUri = _storageUploader.UploadFile(new Uri(_azbatchFileNames.GetStorageJobFolder(jobId)), null).Result;
```

Comment:
IFile interface used in IStorageUploader might need refactored to support open file by path.

Reply:
yes, I took care of it.